Description: Do not enforce atomics support on archs not supporting it.
 armel seems to have no atomic support on opencascade, so make it possible to
 hint the build system not to enforce it.
Author: Tobias Frost <tobi@debian.org>
Forwarded: no, Debian specific.
Last-Update: 2018-05-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,6 +26,12 @@
 endif()
 set (CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Make it possible to ignore if there are no atomics support, needed for armel.
+IF(IGNORE_NO_ATOMICS)
+    ADD_DEFINITIONS(-DIGNORE_NO_ATOMICS)
+ENDIF(IGNORE_NO_ATOMICS)
+
+
 # macro: include patched file if it exists
 macro (OCCT_INCLUDE_CMAKE_FILE BEING_INCLUDED_FILE)
   if (BUILD_PATCH AND EXISTS "${BUILD_PATCH}/${BEING_INCLUDED_FILE}.cmake")
